Index: linux-5.15.34/drivers/hwmon/Kconfig
===================================================================
--- linux-5.15.34.orig/drivers/hwmon/Kconfig
+++ linux-5.15.34/drivers/hwmon/Kconfig
@@ -662,6 +662,16 @@ config SENSORS_G760A
 	  This driver can also be built as a module. If so, the module
 	  will be called g760a.
 
+config SENSORS_G761
+	tristate "GMT G761"
+	depends on I2C
+	help
+	  If you say yes here you get support for Global Mixed-mode
+	  Technology Inc G761 fan speed PWM controller chips.
+
+	  This driver can also be built as a module.  If so, the module
+	  will be called g761.
+
 config SENSORS_G762
 	tristate "GMT G762 and G763"
 	depends on I2C
Index: linux-5.15.34/drivers/hwmon/Makefile
===================================================================
--- linux-5.15.34.orig/drivers/hwmon/Makefile
+++ linux-5.15.34/drivers/hwmon/Makefile
@@ -201,7 +201,7 @@ obj-$(CONFIG_SENSORS_W83L786NG)	+= w83l7
 obj-$(CONFIG_SENSORS_WM831X)	+= wm831x-hwmon.o
 obj-$(CONFIG_SENSORS_WM8350)	+= wm8350-hwmon.o
 obj-$(CONFIG_SENSORS_XGENE)	+= xgene-hwmon.o
-
+obj-$(CONFIG_SENSORS_G761)	+= g761.o
 obj-$(CONFIG_SENSORS_OCC)	+= occ/
 obj-$(CONFIG_PMBUS)		+= pmbus/
 
